packages/opentelemetry-plugin-http/src/enums/AttributeNames.ts

```diff
@@ -38,8 +38,9 @@ export enum AttributeNames {
   IP_UDP = 'IP.UDP',
   HTTP_SERVER_NAME = 'http.server_name',
   HTTP_CLIENT_IP = 'http.client_ip',
+  HTTP_USER_AGENT = 'http.user_agent',
+  HTTP_SCHEME = 'http.scheme',
```

Comment:
If it is not used in the plugin, why is it added here?

Reply:
Even though it's not used in the plugin, it's convenient to reference these attributes when needed instead of redeclaring them somewhere else.
This is the only attribute from the spec which is missing in the list.
I think it's good to have a complete list which follow the spec, instead of users who need it adding this attribute in other places, separated from all other http spec attributes.